utils/createModuleAliases.js

```diff
@@ -21,18 +21,19 @@ const addAlliasesForFolder = (name, aliasObject) => { // components
 };
 
 module.exports = () => {
-  // default aliases
   let moduleAliases = {};
-
   // create specific aliases for locally defined components
   moduleAliases = addAlliasesForFolder('containers', moduleAliases);
   moduleAliases = addAlliasesForFolder('components', moduleAliases);
   moduleAliases = addAlliasesForFolder('pages', moduleAliases);
-
-  moduleAliases['@pages'] = resolve('node_modules/spee.ch-components/lib/pages');
-  moduleAliases['@containers'] = resolve('node_modules/spee.ch-components/lib/containers');
-  moduleAliases['@components'] = resolve('node_modules/spee.ch-components/lib/components');
-
+// default aliases
+  moduleAliases['@pages'] = resolve('node_modules/spee.ch/client/build/pages');
```

Comment:
Can use something like:
```
const BUILD_ROOT = 'node_modules/spee.ch/client/build';
moduleAliases['@containers'] = resolve(`${BUILD_ROOT}/containers`);
```

Also might make sense to look into using helpers or mapping objects routinely (alias/path as key/value)

Reply:
:+1: